utf8.c

```diff
@@ -87,6 +87,34 @@ static Janet jutf8_to_codepoints(int argc, Janet *argv) {
   return janet_wrap_array(a);
 }
 
+static Janet jutf8_from_codepoints(int argc, Janet *argv) {
+  janet_fixarity(argc, 1);
+  JanetView jv = janet_getindexed(argv, 0);
+  const Janet *c = jv.items;
+  utf8proc_ssize_t l = jv.len;
+  utf8proc_ssize_t sl = l * 4;
+  utf8proc_ssize_t bytes_converted = 0;
+  const utf8proc_ssize_t alloca_limit = 64;
+  utf8proc_uint8_t *s = l > alloca_limit ? janet_smalloc(l) : alloca(l);
+  for (utf8proc_ssize_t i = 0; i < l; i++) {
+    utf8proc_uint8_t r[4];
+    utf8proc_int32_t cp = janet_unwrap_integer(c[i]);
```

Comment:
I think you need to check it is a number before unwrapping.

Reply:
Indeed, thank you I will fix it.